### PR TITLE
Simple form validation

### PR DESCRIPTION
**Pull request by katfogy**
The follow are the features:
- The content of the email field has to be in lower case
- When the user submits the form, you check if the email is in lower case
- If the validation is OK, the form is sent.
- If the validation is not OK, It show an error message to the user near the submit button informing them of the error and the form is not sent.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @katfogy, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that should be built using peer programming, kindly make sure all the student's in the group contribute to this PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -171,4 +179,53 @@ modal.addEventListener('click', (e) => {
...
+    emailError.style.display = 'block';
+    ResetError(email, emailError);
+  } else if (message.value === '') {
+    message.style.border = '2px solid red';
+    messageError.textContent = 'Message is Required';
+    messageError.style.display = 'block';
+    ResetError(message, messageError);
+  } else if (message.value.length > 500) {
+    message.style.border = '2px solid red';
+    messageError.textContent = 'Message Body cannot Exceed 500 Characters';
```
Comment (brainconnect93) [inline index.js]: - [ ] Hello @katfogy, great effort working on the form validation, currently i noticed, the validation is not working as expecting. According to the requirement, user should not be able to submit the form if there is an `uppercase` letter in the `email address` and an error you pop around the button or the input form stating that the user should make his/her email `lowercase.`
I notice i can submit the form with uppercase letter in my email. Kindly fix this before you resubmit for review

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @katfogy, 👋

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- Pair programming activity ✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @katfogy, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**